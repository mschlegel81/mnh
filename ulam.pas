PROGRAM ulam;
USES mypics;

FUNCTION numberOfFactors(CONST n:longint):longint;
  VAR k:longint;
  begin
    k:=2;
    result:=1;
    while k*k<=n do begin
      while (n mod k)=0 do begin
        inc(result);
        n:=n div k;
      end;
      inc(k);
    end;
  end;
  
FUNCTION numberAt(x,y:longint):longint;
  begin
    
  end;