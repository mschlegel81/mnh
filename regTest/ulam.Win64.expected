          XX                                        XX          XX                      XX          XX                  
                   XX                XX          XX    XX          XX                                        XX         
    XX                XX    XX                XX          XX    XX                                                    XX
 XX                XX    XX                                                                XX                XX         
                      XX    XX                XX          XX                                  XX                XX      
                                                       XX          XX    XX          XX                                 
    XX                                              XX          XX                      XX          XX    XX            
 XX                XX          XX                                  XX    XX                XX                           
    XX          XX    XX                XX    XX                XX                XX                                  XX
 XX    XX                                  XX                                  XX          XX                XX    XX   
          XX                XX          XX                      XX                XX                                    
             XX          XX                            XX          XX    XX          XX    XX    XX                     
    XX    XX                      XX                XX                            XX    XX          XX                XX
                   XX                XX          XX    XX                                              XX               
                                                          XX    XX                XX          XX          XX    XX      
 XX          XX          XX          XX    XX          97                      XX          XX    XX                     
                                                          61    59          XX                                          
       XX                      XX          XX    37                31    89    XX                XX    XX    XX         
          XX    XX    XX    XX    XX    XX    67    17          13                      XX                            XX
                                                       _5    _3    29                                  XX               
                                        XX          19       _2 11    53    XX    XX          XX    XX    XX            
                   XX                      XX    41    _7                                                               
                                              71          23                                                            
             XX          XX    XX          XX    43          47          83    XX          XX          XX    XX         
    XX                XX          XX          73                79                XX    XX          XX                  
                                                 XX                                  XX                                 
    XX                            XX    XX                XX          XX          XX                      XX    XX      
 XX    XX                XX          XX                                  XX                      XX                XX   
          XX                XX                XX          XX    XX                                                      
                                           XX    XX          XX                XX          XX                XX         
          XX    XX    XX    XX                            XX    XX                XX                XX    XX          XX
                   XX          XX                                  XX    XX                                             
    XX          XX    XX                XX                XX          XX    XX                                  XX      
       XX    XX                      XX                            XX                      XX                XX         
                                        XX    XX          XX    XX                            XX                XX      
                               XX          XX                XX                XX                            XX    XX   
                      XX    XX                XX                                                    XX                  
 XX          XX                      XX                                  XX          XX                XX               
                                                                                        XX                XX          XX
 XX          XX    XX          XX                XX                                  XX                                 
